pandas/indexes/multi.py

```diff
@@ -368,15 +368,26 @@ def copy(self, names=None, dtype=None, levels=None, labels=None,
         ``deep``, but if ``deep`` is passed it will attempt to deepcopy.
         This could be potentially expensive on large MultiIndex objects.
         """
+        if names is not None and name is not None:
+            raise TypeError("Can only provide one of `names` and `name`")
+        if names is None:
+            names = name
```

Comment:
Here I assume that the user has passed-in a list for `name`. A scalar entry for a `MultiIndex` doesn't make any sense.